### ISSUE TITLE
concurrent/timer_task should load its dependencies

### ISSUE THREAD

**Issue by kares** [ruby-concurrency/concurrent-ruby#474]
currently: `require 'concurrent/timer_task'` : 
```
  NameError: uninitialized constant Concurrent::RubyExecutorService
/opt/local/rvm/gems/jruby-9.0.4.0@jdbc/gems/concurrent-ruby-1.0.0-java/lib/concurrent/timer_task.rb:153:in `<module:Concurrent>'
/opt/local/rvm/gems/jruby-9.0.4.0@jdbc/gems/concurrent-ruby-1.0.0-java/lib/concurrent/timer_task.rb:8:in `<top>'
```

**Comment by jdantonio** [comment]
Thank you for reporting this. I'll work on it next week.

**Comment by jdantonio** [comment]
Sorry it took me so long to work on this. After the release of 1.0 this bug got lost in the shuffle. My apologies. I've submit a fix in PR #505. That fix should be a part of the upcoming 1.0.1 release.
